x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/MetadataAttribute.java

```diff
@@ -25,14 +21,14 @@ public class MetadataAttribute extends TypedAttribute {
 
     private static final Map<String, Tuple<DataType, Boolean>> ATTRIBUTES_MAP = Map.of(
         "_version",
-        tuple(DataTypes.LONG, true),
+        tuple(DataTypes.LONG, false), // _version field is not searchable
         "_index",
         tuple(DataTypes.KEYWORD, true),
         "_id",
-        tuple(DataTypes.KEYWORD, false)
+        tuple(DataTypes.KEYWORD, false) // actually searchable, but fielddata access on the _id field is disallowed by default
     );
 
-    private final boolean docValues;
```

Comment:
`docValues` was added for future `_id` support. However, `_id` support matches by name, not by checking if the attribute has or not `docValues`; so scrapped and replaced with `searchable`.